src/content/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-12-8-0.mdx

```diff
@@ -0,0 +1,53 @@
+---
+subject: Node.js agent
+releaseDate: '2024-11-19'
+version: 12.8.0
+downloadLink: 'https://www.npmjs.com/package/newrelic'
+security: []
+bugs: ["Corrected expected shape of user input in get Prompt()"]
+features: ["Added new configuration options to add custom tags (labels) to logs","Moved `entity.guid`, `entity.name`, `entity.type`, and `hostname` to `common.attributes` on logs payload instead of in every log message","Added `labels.enabled` and `labels.exclude` to config defaults","Added support for Bedrock Claude 3.5 invoke model user request"]
```

Comment:
```suggestion
features: ["Added new configuration options to add custom tags (labels) to logs","Moved `entity.guid`, `entity.name`, `entity.type`, and `hostname` to `common.attributes` on logs payload instead of in every log message","Added support for Bedrock Claude 3.5 invoke model user request"]
```

Reply:
to whomever reviews this, can you please accept this, we had to tweak our changelog and this was the update